go.mod

```diff
@@ -205,7 +205,6 @@ require (
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
 	github.com/hashicorp/go-retryablehttp v0.7.7 // indirect
-	github.com/hashicorp/terraform-json v0.14.0 // indirect
```

Comment:
I expect this will cause problems [too][1], but as discussed in the other thread, I'm not sure how to avoid it.

[1]: https://github.com/openshift/installer/pull/9002#discussion_r1757636897